OlxProject/Tools/driver.rb

```diff
@@ -1,13 +1,16 @@
 require 'rubygems'
 require 'selenium-webdriver'
 
-public def StartTest
-  $driver = Selenium::WebDriver.for :chrome
-  $driver.manage.timeouts.implicit_wait = 5
-  $driver.manage.window.maximize
-  $driver.navigate.to "https://www.olx.ua/"
+def driver
+  $driver ||= Selenium::WebDriver.for :chrome
 end
 
-public def DriverClose
-  $driver.quit
+def start_test
+  driver.manage.timeouts.implicit_wait = 10
+  driver.manage.window.maximize
+  driver.navigate.to "https://www.olx.ua/"
```

Comment:
what if we want to re-use driver class for other site? we always need to start at olx?

Reply:
In this case I test only olx.ua, but sure, I can make this method with parameter like
def start_test(web_site)
   ...
   driver.navigate.to web_site
end
then we can call this method with parameter like
start_test("https://www.olx.ua/")